CHANGELOG.md

```diff
@@ -6,6 +6,15 @@
 
 - Add new User Feedback API ([#4286](https://github.com/getsentry/sentry-java/pull/4286))
     - We now introduced Sentry.captureFeedback, which supersedes Sentry.captureUserFeedback
+- Add Sentry Log Feature ([#4372](https://github.com/getsentry/sentry-java/pull/4372))
+    - The feature is disabled by default and needs to be enabled by:
+        - `options.getLogs().setEnabled(true)` in `Sentry.init` / `SentryAndroid.init`
+        - `logs.enabled=true` in `sentry.properties`
+        - `sentry.logs.enabled=true` in `application.properties`
+        - `sentry.logs.enabled: true` in `application.yml`
```

Comment:
manifest or no manifest? 👀

Reply:
Ah yeah we should add it to manifest as well.

Reply:
Added in https://github.com/getsentry/sentry-java/pull/4395